trunner/device.py

```diff
@@ -1,18 +1,19 @@
 # Imports of available runners
 from .runners import HostRunner, QemuRunner, IMXRT106xRunner, IMXRT117xRunner, STM32L4Runner
 
-from .config import PHRTOS_PROJECT_DIR, DEVICE_SERIAL_USB, PHOENIXD_PORT
+from .config import PHRTOS_PROJECT_DIR, DEVICE_SERIAL_USB, PHOENIXD_PORT, QEMU_TARGETS
 
 
 QEMU_CMD = {
```

Comment:
not directly related to this PR but probably should be (sooner or later) moved to `QemuRunner` class

Reply:
Do You mean `QEMU_CMD` should be moved?

Reply:
It should probably internal to `QemuRunner` class (so it can be instantiated by the target directly

Reply:
Do You mean sth I've done now, yes?